style.css

```diff
@@ -0,0 +1,181 @@
+body{
+    color: #7069f3;
+}
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.hamburger{
+    position : absolute;
+    z-index: 2;
+    right: 0;
+}
+
+h1{
+    text-align: center;
+    padding: .1em 0;
+}
+
+.overflow-auto{
+    overflow-y: auto;
+    scrollbar-width: thin;      
+    scrollbar-color: #7069f3 #fff;  
+}
+.overflow-auto::-webkit-scrollbar {
+    width: 2px;               /* width of the entire scrollbar */
+  }
+.overflow-auto::-webkit-scrollbar-thumb {
+background-color: #fff;    /* color of the scroll thumb */
+}
+.overflow-auto::-webkit-scrollbar-track {
+background: #7069f3;        /* color of the tracking area */
+}
+
+#new-note{
```

Comment:
Don't style IDs to style - they have very high specificity. So in future this selector could override some other selector and it would be hard to debug n fix. Always try to use single class as the your selectors